swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -35,63 +35,77 @@ Follow the instructions given in this section to develop the service.
     $ bal new sending-a-message-to-a-service
     ```
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+2. Remove the generated content in the `main.bal` file and open the [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code.
```

Comment:
We aren't really using the API designer here, right? What if we combine 2 and 3 and say open the diagram view and add two configurable variables...

Reply:
And mention the HTTP API designer where it is actually used (currently point 5).

Reply:
I feel like adding "open diagram" in a step would be more make sense because we need a diagram view for every step.